ebook/en/content/030-github-marksdown-cheatsheet.md

```diff
@@ -0,0 +1,243 @@
+GitHub Markdown Cheatsheet
+===================
+
+- - - - 
+# Heading 1 #
+
+    Markup :  # Heading 1 #
+
+    -OR-
+
+    Markup :  ============= (below H1 text)
+
+## Heading 2 ##
```

Comment:
> I believe that the way the tags are defined here, it will break the eBook semantics, can you maybe restructure this and wrap the examples in code snippets?

The above would apply for all heading examples

Reply:
The Heading1, Heading2, Heading3, and Heading4 give a preview of how the headings would look like.
I've added subheadings for the remaining docs

@bobbyiliev

Reply:
Here is how the generated eBook looks like:

- The H1 is picked up as a separate chapter:
<img width="730" alt="image" src="https://user-images.githubusercontent.com/21223421/195797116-57b8518a-d5db-4723-9fac-295058c74384.png">

- The Markdown chapter itself is displayed as completely empty:
<img width="601" alt="image" src="https://user-images.githubusercontent.com/21223421/195797290-c9d7cafc-a36a-4d82-ac4e-85f64c772fff.png">